linchpin/provision/roles/dependencies/tasks/beaker_deps.yml

```diff
@@ -1,9 +1,3 @@
 - name: Install pypi dependencies of beaker
   pip:
-    name: "{{ bkr_pkg }}"
-  with_items:
-    - 'beaker-client>=23.3'
-    - 'python-krbV'
-    - 'bottle'
-  loop_control:
-    loop_var: bkr_pkg
+    name: ['git+https://github.com/beaker-project/beaker.git@develop#subdirectory=Client/', 'python-krbV', 'bottle']
```

Comment:
I can see that you are trying to get beaker from Develop branch. Develop branch contains Beaker 27.
I replaced python-krbV with python-gssapi

Reply:
I could not find a tag specific to beaker client that is the reason I have used to develop branch code. 
Further, does the above mean python-krbV is no longer needed?

Reply:
Beaker client doesn't require python-krbV anymore. python-gssapi is sufficient for beaker client / common.

Reply:
+ I think you should consume our tags.. Not blindly install develop

Reply:
I completely agree with you. 
However, could share the link/tag which contains python3 patch
the GitHub repo seems confusing. 
Thank you.

Reply:
Hello @samvarankashyap.
Python3 patch is not tagged (yet). I'm working on it, we have to finish Beaker 27 changes. Then I can provide you tag. I'm about to release 26.6. And we are close to 27 - We are performing tests in stage.